woocommerce/woocommerce-ios
# Add padding between email and Continue button

Closes #3309
Closes #3330
Ref: wordpress-mobile/WordPressAuthenticator-iOS#544

#3330 was suggested as an alternative in case #3309 was not possible.

The actual work is done in WordPressAuthenticator. This PR only points the dependency to the branch in WPAuthnticator that implements the solution.

Please note that the actual amount of padding is decided by WPAuthenticator. Having more granular control over the amount of padding would be pretty complex.

## Changes
| Before | After | 
| ---- | ---- |
| <img src="https://user-images.githubusercontent.com/2722505/102027755-446f1a80-3de1-11eb-9c28-547c32dfc90d.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/102027759-4802a180-3de1-11eb-8e0d-b9b46c02640b.png" width="350"/> |

## How to test
* Checkout the branch. Run `bundle exec pod install`
* Log out if necessary. Attempt to login in with a WordPress.com account. Notice the padding between the email address field and the continue button

Update release notes:
